tests/st/calicoctl/test_default_pools.py

```diff
@@ -0,0 +1,168 @@
+# Copyright (c) 2015-2016 Tigera, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import functools
+import logging
+
+import netaddr
+import yaml
+from nose_parameterized import parameterized
+
+from tests.st.test_base import TestBase
+from tests.st.utils.docker_host import DockerHost, CLUSTER_STORE_DOCKER_OPTIONS
+from tests.st.utils.exceptions import CommandExecError
+from tests.st.utils.utils import get_ip, wipe_etcd, retry_until_success
+
+_log = logging.getLogger(__name__)
+_log.setLevel(logging.DEBUG)
+
+
+class TestDefaultPools(TestBase):
+    @classmethod
+    def setUpClass(cls):
+        # First, create a host to run things in
+        cls.hosts = []
```

Comment:
Do we need hosts here - in some places you assume only one host and others (cleanup) that there are multiple hosts.  Maybe just get rid of it and assume this is a one-host suite.